backend/.env.sample

```
@@ -16,4 +16,9 @@ CORS_EXPOSE_HEADERS=Content-Length
 CORS_ALLOW_CREDENTIALS=true
 CORS_MAX_AGE=43200
 ACCESS_DURATION=10
-REFRESH_DURATION=4320
\ No newline at end of file
+REFRESH_DURATION=4320
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REFRESH_DURATION key should go before the SECRET_TOKEN key

(UnorderedKey)